modules/services/model/src/main/java/au/com/shiftyjelly/pocketcasts/models/db/dao/EpisodeDao.kt

```diff
@@ -363,11 +363,11 @@ abstract class EpisodeDao {
         """
         SELECT * 
         FROM episodes
-        WHERE episodes.last_playback_interaction_date IS NOT NULL AND episodes.last_playback_interaction_date < :fromEpochMs
+        WHERE episodes.last_playback_interaction_date IS NOT NULL AND episodes.last_playback_interaction_date < :fromEpochMs AND episodes.last_playback_interaction_date != 0
```

Comment:
If an episode is deleted from listening history this is set to 0 so this ignores that episode.